scripts/genvp.py

```diff
@@ -1391,7 +1391,7 @@ def parseVersionInfo(self, xml):
 
     def parseExtensionInfo(self, xml):
         self.extensions = dict()
-        for ext in xml.findall("./extensions/extension"):
+        for ext in xml.findall("./extensions/extension[@supported='vulkan']"):
             # Only care about enabled extensions
             if ext.get('supported') == "vulkan":
```

Comment:
Now this explicit if-check on the attribute value can be eliminated as the XPath filters out any other possibilities.

Reply:
Good catch, thanks!